codegen/cpp/fletchgen/src/fletchgen/options.cc

```diff
@@ -79,8 +79,11 @@ bool Options::Parse(Options *options, int argc, char **argv) {
                  "  bm : Bus maximum burst size.\n"
                  "Currently supports only one top-level bus specification. Default: \"64,512,64,8,1,16\"");
 
-  app.add_flag("--axi", options->axi_top,
-               "Generate AXI top-level template (VHDL only).");
+  app.add_flag("--mmio64", options->mmio64, "Use a 64-bits AXI4-lite MMIO data bus instead of 32-bits.");
```

Comment:
Maybe you can also add a `--mmio-addr-offset` option so I can add the OPAE registers without having to manually modify the addresses of all generated registers in the yaml file.

Reply:
On the software side this would be fixed in the platform-specific lib?

Reply:
Yes. Example here: https://github.com/teratide/fletcher-opae/commit/8e35f1dbfb3a51fe70205420e84787c7d7973eb4

Reply:
Or maybe I should just use the `--regs` option? That would work.

Reply:
I added this